packages/office-ui-fabric-react/src/components/SwatchColorPicker/examples/SwatchColorPicker.Basic.Example.tsx

```diff
@@ -57,7 +57,6 @@ export class SwatchColorPickerBasicExample extends React.Component<any, IBasicSw
         </div>
         <SwatchColorPicker
           // tslint:disable:jsx-no-lambda
-          onCellHovered={(id, color) => this.setState({ previewColor: color! })}

```

Comment:
I looked at where the [onCellHovered callback](https://github.com/OfficeDev/office-ui-fabric-react/blob/92ba8325a3f01c050da0d9e50effb1c399e39663/packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.base.tsx#L299) is triggered is SwatchColorPicker.base.tsx and it doesn't look like anything with color is incorrect so I thought the example was just not using it properly... do you see anything with the callback that is incorrect?

Reply:
I didn't investigate. The only thing I found if it's removed it's working as intended. But better instead of removing we need to look at what is wrong with its use. If it's not used properly we should modify the example in demoing how to use it instead of removing.

Reply:
Ok from my investigation so far, I haven't found anything wrong with the callback - it seems very straightforward but I'll look into it further!

Reply:
Spoke with David and learned that this is all due to the state not being set properly in componentWillReceiveProps - pushed new commit with these changes.

Reply:
Thanks for digging into it and for the fix.

Reply:
@aneeshack4 
While waiting for a deploy PR link to test the change I was wondering why 2 change files nedeed? Can you combine in just one?